src/renderer/pages/Staking/Operations/Bond/Bond.tsx

```diff
@@ -232,18 +233,22 @@ export const Bond = () => {
             onGoBack={goToPrevStep}
             {...explorersProps}
           >
-            {isAlertOpen && (
-              <Alert title={t('staking.confirmation.hintTitle')} onClose={toggleAlert}>
-                <Alert.Item>{t('staking.confirmation.hintRewards')}</Alert.Item>
-                <Alert.Item>
+            <StakingPopover labelText={t('staking.confirmation.hintTitleStartStaking')}>
```

Comment:
It's not necessary to have that component at all but I think it might be nice to hide repeated `Popover` code for all of the staking components. I can make it more like Alert component in this case (or we still can just delete it) 

I have this structure from matrixPopover, but I can rearrange `<p>` it inside of `<li>`